packages/gridproxy_client/src/modules/nodes.ts

```diff
@@ -116,6 +116,11 @@ export class NodesClient extends AbstractClient<NodesBuilder, NodesQuery> {
     return res.json();
   }
 
+  public async nodesOwnedBy(twinId: number): Promise<Node[]> {
```

Comment:
why do we create a new method for it when we can update the `list` method with one extra query?

Reply:
`ownedBy` was added as an extra query parameter to `list` and the `nodesOwnedBy` function was removed in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1572/commits/79743fa92f4b25a191aae31badddf6f5e7ef052c